src/test/aws/construct/static-asset-deployment.test.ts

```diff
@@ -24,89 +24,135 @@ const testStackProps = {
   stageContextPath: 'src/test/aws/common/cdkEnv',
 }
 
-class TestCommonStack extends CommonStack {
-  declare props: TestStackProps
-
-  constructor(parent: cdk.App, name: string, props: cdk.StackProps) {
-    super(parent, name, props)
-
-    this.construct = new TestStaticAssetDeployment(this, testStackProps.name, this.props)
-  }
+// get props
+function createTestStack<P extends CommonStackProps, T extends CommonConstruct<P>>(
```

Comment:
this can be a common function that can be used to test stacks with a single single construct, instead of creating `TestStacks`